src/config.rs

```diff
@@ -426,9 +426,111 @@ pub struct DbOptions {
     ///
     /// Default: no TTL (insertions will remain until deleted)
     pub default_ttl: Option<u64>,
+
+    pub max_unflushed_memtable: usize,
```

Comment:
is this meant to be in this pr?

Reply:
yup, the build was running into issues since this was not part of the init values that are being set up. I am open for suggestions.

Reply:
It sounds like you had an issue rebasing. This code was deleted in #303. It's no longer in main. Please remove it.